docs/glossary/glossary.md

```diff
@@ -9,11 +9,11 @@ Here's a list of the most commonly used terms:
+| Function            | A simple code snippet that you can run without provisioning or managing servers. It implements the exact business logic you define. A Function is based on the Function custom resource and can be written in either Node.js or Python. A Function can perform a business logic of its own. You can also bind it to an instance of a service and configure it to be triggered whenever it receives a particular event type from the service or a call is made to the service's API. Functions are executed only if they are triggered by an event or an API call. |       |
+| Microservice        | An architectural variant for extensions or applications, where you separate the tasks in smaller pieces that interact with each other as loosely coupled, independently deployable units of code. A failing microservice should not cause your whole application to fail. Microservices are packed in a container that is always running; it's idling if there is no load. The microservice should always be reachable even when the pods move around. Microservices typically communicate through APIs.                                                                                                                                                                   |       |
 | Role                | Access to every cluster is managed by the roles assigned. Roles give the assigned users a different level of permissions suitable for different purposes.                                                                                                                                                                                                                                                                                                                  |       |
-| Namespace           | Namespaces organize objects in a cluster and provide a way to divide cluster resources. This way, several users can share a cluster but have access only to resources within the Namespace they have permissions for. This increases the security and organization of your cluster by dividing it into smaller units. Access to Namespaces in the Kyma environment depends on your permissions.                                                                                     |       |
+| Namespace           | Namespaces organize objects in a cluster and provide a way to divide cluster resources. This way, several users can share a cluster but have access only to resources within the Namespace they have permissions for. This increases the security and organization of your cluster by dividing it into smaller units. Access to Namespaces in the Kyma environment depends on your permissions.                                                                                     | [Kubernetes Namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) |
```

Comment:
The last sentence kind of feels like a repetition of `but have access only to resources within the Namespace they have permissions for.`, but it's nice and concise and I wouldn't like to see it go. 
How about this? 
```suggestion
| Namespace           | Namespaces organize objects in a cluster and provide a way to divide cluster resources. This way, several users can share a cluster but have access to resources restricted to specified Namespaces only. This increases the security and organization of your cluster by dividing it into smaller units. Access to Namespaces in the Kyma environment depends on your permissions.                                                                                     | [Kubernetes Namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) |
```
It's just a suggestion, it doesn't bother me much, so if you'd rather keep the original, I'm fine with it too :)

Reply:
Shorter is (almost) always better!